File: Jarvis.Web/MG.Jarvis.Api.Extranet/ViewModel/ContactDetailsViewModel.cs
namespace MG.Jarvis.Api.Extranet.ViewModel
{
    public class ContactDetailsViewModel : BaseViewModel
    {
        public int ContactId { get; set; }
        public bool IsPrimary { get; set; }
        public string ContactPerson { get; set; }
        public string EmailAddress { get; set; }
        public string ContactNumber { get; set; }
        public int DesignationId { get; set; }
        public bool IsDeleted { get; set; }

    }
}

File: Jarvis.Web/MG.Jarvis.Web/ClientApp/src/app/extranet/viewmodel/hoteldetailsviewmodel/contactdetailsviewmodel.ts
import { IObjectWithState } from '../../../common/iobjectwithstate';
import { ObjectState } from '../../../common/enums';

export class ContactDetailsViewModel implements IObjectWithState {
    objectState: ObjectState;
    contactId: number;
    isPrimary: boolean;
    contactPerson: string;
    designationId: number;
    emailAddress: string;
    contactNumber: string;
}
